### ISSUE TITLE
Hello, I used your model to train my own data set, one-dimensional signal data set, but when I trained, I found that the accuracy rate was very low, so I did not move 0.08. Why?

### ISSUE THREAD

**Issue by 1448643857** [lmartak/distill-nn-tree#3]
thank u very much

**Comment by 1448643857** [comment]
Epoch 1/500
24000/24000 [==============================] - 718s 30ms/step - loss: 14.0100 - acc: 0.0835 - val_loss: 13.9278 - val_acc: 0.0883
Epoch 2/500
24000/24000 [==============================] - 719s 30ms/step - loss: 13.9117 - acc: 0.1274 - val_loss: 13.9540 - val_acc: 0.0847
Epoch 3/500
24000/24000 [==============================] - 718s 30ms/step - loss: 13.7736 - acc: 0.2407 - val_loss: 14.1120 - val_acc: 0.0883
Epoch 4/500
24000/24000 [==============================] - 716s 30ms/step - loss: 13.4118 - acc: 0.3258 - val_loss: 14.5342 - val_acc: 0.0895
Epoch 5/500
24000/24000 [==============================] - 719s 30ms/step - loss: 12.7447 - acc: 0.3750 - val_loss: 14.8735 - val_acc: 0.0938
Epoch 6/500
24000/24000 [==============================] - 718s 30ms/step - loss: 11.8372 - acc: 0.3967 - val_loss: 15.1256 - val_acc: 0.0912
Epoch 7/500
24000/24000 [==============================] - 720s 30ms/step - loss: 10.8221 - acc: 0.4143 - val_loss: 15.2756 - val_acc: 0.0888
Epoch 8/500
24000/24000 [==============================] - 719s 30ms/step - loss: 9.8527 - acc: 0.4263 - val_loss: 15.3327 - val_acc: 0.0922
Epoch 9/500
24000/24000 [==============================] - 718s 30ms/step - loss: 9.0603 - acc: 0.4356 - val_loss: 15.3739 - val_acc: 0.0935
Epoch 10/500
24000/24000 [==============================] - 718s 30ms/step - loss: 8.4028 - acc: 0.4433 - val_loss: 15.3867 - val_acc: 0.0940
Epoch 11/500
24000/24000 [==============================] - 720s 30ms/step - loss: 7.9059 - acc: 0.4503 - val_loss: 15.4083 - val_acc: 0.0947
Epoch 12/500
24000/24000 [==============================] - 719s 30ms/step - loss: 7.4681 - acc: 0.4538 - val_loss: 15.4072 - val_acc: 0.0932
Epoch 13/500
24000/24000 [==============================] - 719s 30ms/step - loss: 7.1606 - acc: 0.4591 - val_loss: 15.4000 - val_acc: 0.0897
Epoch 14/500
24000/24000 [==============================] - 720s 30ms/step - loss: 6.9462 - acc: 0.4662 - val_loss: 15.3871 - val_acc: 0.0942
Epoch 15/500
24000/24000 [==============================] - 722s 30ms/step - loss: 6.7773 - acc: 0.4675 - val_loss: 15.3829 - val_acc: 0.0920
Epoch 16/500
24000/24000 [==============================] - 719s 30ms/step - loss: 6.6513 - acc: 0.4730 - val_loss: 15.3680 - val_acc: 0.0892
Epoch 17/500
24000/24000 [==============================] - 718s 30ms/step - loss: 6.5321 - acc: 0.4774 - val_loss: 15.3624 - val_acc: 0.0913
Epoch 18/500
24000/24000 [==============================] - 714s 30ms/step - loss: 6.4380 - acc: 0.4801 - val_loss: 15.3570 - val_acc: 0.0942

Like this

**Comment by lmartak** [comment]
As your training accuracy increases and training loss decreases, this looks like a problem of generalization related to the data you're using. As the problem is not with this code but with how its used, I'm closing the issue.
